src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/MethodWithGCInfo.cs

```diff
@@ -42,6 +44,8 @@ public void SetCode(ObjectData data)
 
         public MethodDesc Method => _method;
 
+        public List<ISymbolNode> TypesToPreload => _typesToPreload;
```

Comment:
Some day we may want to place non-Types in this list. I'd like to see the name reflect that these are general purpose fixups, not just types.